courses/machine_learning/deepdive/05_review/5_train.ipynb

```diff
@@ -19,19 +19,20 @@
   },
```

Comment:
<p>I guess this should be "Train on Cloud AI Platform"</p><p>and CMLE -&gt; CAIP</p><p>and in body of text, Cloud ML Engine -&gt; Cloud AI Platform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/730/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done!</p>